build-parent/pom.xml

```diff
@@ -493,13 +493,11 @@
                                             <exclude>io.quarkus:quarkus-test-*</exclude>
                                             <exclude>io.rest-assured:*</exclude>
                                             <exclude>org.assertj:*</exclude>
-                                            <exclude>org.junit.jupiter:*</exclude>
                                         </excludes>
                                         <includes>
                                             <include>io.quarkus:quarkus-test-*:*:*:test</include>
                                             <include>io.rest-assured:*:*:*:test</include>
                                             <include>org.assertj:*:*:*:test</include>
-                                            <include>org.junit.jupiter:*:*:*:test</include>
```

Comment:
We shouldn't do that, we should override the rule for the artifacts needing it. Otherwise we will definitely get it wrong (we added this rule for a good reason).

I understand it wasn't a priority until now but let's not merge it this way. Maybe @famod can help.

Reply:
I can have a look at the branch later this week (or more or less shortly after it was merged).

Reply:
I updated it, I think it should be ok now.

Reply:
So you basically don't check transitively for junit anymore, right?
I wonder if we should narrow it down even more by only overriding `searchTransitive` to false in _specific_ modules?

Reply:
Because testing is implemented in core/deployment every -deployment artifact would need it.